src/components/Faq/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {FaqStyleTitle, FaqStyleContent} from './styles'
+
+class FAQ extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      clickedOn: false
+    }
+    this.handleClick = this.handleClick.bind(this);
+  }
+  
+  handleClick() {
+    // var content = this.nextElementSibling;
+    // console.log("CONTENT STYLE", content.style.maxHeight);
+    this.setState({
+      clickedOn: !this.state.clickedOn
+    });
+  }
+  render() {
+    const {title, answer} = this.props;
+    const questionSelected = this.state.clickedOn;
```

Comment:
Instead of creating a new variable here, you can use the ES6 destructuring feature: 

`const { clickedOn } = this.state;`